docs/Makefile

```diff
@@ -1,4 +1,42 @@
-# Minimal makefile for Sphinx documentation
+all:  pbench-agent.html test-rpms.html
+
+everything: all pbench-agent.pdf help
+
+%.html:	%.org
+	emacs -batch -l batch-export.el --eval "(batch-org-export-as 'html)" $<
+
+%.pdf:	%.tex
+	latexmk --shell-escape -pdf -xelatex $<
```

Comment:
@ndokos, this command fails for me with missing `*.sty` files that `latexmk` prompts me for during the build, interactively.

Reply:
I'll check.

Reply:
The Org mode LaTeX exporter adds a bunch of packages to the export which need to be installed on whatever box you are doing the LaTeX processing.  The question is whether we want to worry about producing PDF.

Reply:
So ... does this mean we should just remove the `%.pdf` target? Write some instructions about installing the packages to actually generate it?

Reply:
Yes, but I'd rather do that once we decide how we'll be creating documentation in the future.